sources/packages/api/src-sql/Applications/Add-col-program-year-id.sql

```diff
@@ -0,0 +1,9 @@
+-- Create program_year_id
+ALTER TABLE
+    sims.applications
+ADD
+    COLUMN IF NOT EXISTS program_year_id INT REFERENCES sims.program_years(id) ON DELETE
```

Comment:
I believe that this column should not allow NULL values. Is there a scenario where it could be null?

Reply:
Yeah when the column is in Draft state.

Reply:
Why it will be null in the draft state?

Reply:
Its not passed in Draft state, so it will be null. That is the part of the requirement.

Reply:
Where is it in the requirement?

Reply:
"Along side with Application Submission send the Program Year ID"
 
 During submission , program Year id needs to be sent and saved. Not during draft state